draft-ietf-httpbis-resumable-upload.md

```diff
@@ -208,6 +208,8 @@ If the request completes successfully but the entire upload is not yet complete,
 
 If the request includes an `Upload-Complete` field value set to true and a valid `Content-Length` header field, the client attempts to upload a fixed-length resource in one request. In this case, the upload's final size is the `Content-Length` field value and the server MUST record it to ensure its consistency.
 
+The request content MAY be empty. If the `Upload-Complete` header field is then set to true, the client intends to upload an empty entity. If the `Upload-Complete` header field is then set to false, the client wants to obtain the upload resource's URL before transferring data. This is useful when the client is not able to retrieve the resource URL in an informational response.
```

Comment:
While the statement is correct, I don't think we want to promote the idea that 1xx can fail, how about something like

```suggestion
The request content MAY be empty. If the `Upload-Complete` header field is then set to true, the client intends to upload an empty entity. An `Upload-Complete` header field is set to false is also valid. This can be used to create an upload resource URL before transferring data, which can save client or server resources. Since informational responses are optional, this technique provides another mechanism to learn the URL, at the cost of an additional round-trip before data upload can commence.
```

Reply:
Good suggestion, thank you! I didn't want to imply that the transmission of a 1xx can fail, but rather wanted to highlight situations where the client is incapable of receiving them because the HTTP interface does not expose them. But your change expresses that better.